convert_gvf_to_vcf/convertGVFtoVCF.py

```diff
@@ -227,11 +227,35 @@ def format_sample_values(sample_name_dict_format_kv, list_of_sample_names):
     sample_format_value_tokens = []
```

Comment:
What I would suggest instead is something like:
Assuming you have `list_of_sample_names` and `list_of_format_key`
```python
for sample in list_of_sample_names:
    if sample in sample_name_dict_format_kv:
        sample_format_value_tokens.append(':'.join([sample_format_value_tokens[sample].get(key, '.') for key in list_of_format_key]))
    else:
        sample_format_value_tokens.append(':'.join(['.' for key in list_of_format_key]))
```